lib/msf/core/exploit/remote/http/http_cookie.rb

```diff
@@ -30,7 +30,7 @@ def initialize(name, value = nil, **attr_hash)
             end
 
             attr_hash.each_pair do |k, v|
-              if k == 'max-age'
+              if k == 'max-age'.to_sym
```

Comment:
do you need to call `to_sym` on this? and if so why not just check if it's equal to `:max-age` instead?

Reply:
That's a syntax error, you'd want `:'max-age'`

Reply:
ohhh well that explains why it was done the way that it is :sweat_smile: